### PR TITLE
[Beat] Add Innovation OCR Receipt Recognition (OCR Struk)

### PR DESCRIPTION
**Pull request by sonadztux**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

There's no http request handler for new innovation.

## What is the new behavior?

For this week, just added a http request handler for new innovation OCR receipt (OCR struk)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Screenshot at 2021-11-22 13-06-55

### CODE REVIEW

File: backend/database/seed.go [hunk 1]
```diff
@@ -173,16 +173,16 @@ func seedService(model *models.Model) {
...
-			ShortDescription:   "Rotten Fruit Detection Description",
-			LongDescription:    "Rotten Fruit Detection Looonng Descriptiooonnn",
-			SpecialInstruction: "Special Instruction for Rotten Fruit Detection",
+			Slug:               "ocr-receipt",
+			Thumbnail:          "ocr-receipt.png",
+			AccessKey:          "",
+			Token:             	"",
+			Timestamp:          "",
+			ShortDescription:   "OCR Receipt Recognition Description",
+			LongDescription:    "OCR Receipt Recognition Looonng Descriptiooonnn",
```
Comment (krismp) [inline backend/database/seed.go:184]: Toloooooooong

:trollface:

Comment (sonadztux) [reply backend/database/seed.go:184]: @krismp hahaha. Emang belum ada copywriting-nya dari product mas

Comment (krismp) [reply backend/database/seed.go:184]: ATM dari sini aja dulu @sonadztux:

https://github.com/nodefluxio/weber/pull/128/files#diff-39838703e8840b770c775df5b50257ce6cb10bd266c10c2beb7ae65fb5e5db17R71

Comment (sonadztux) [reply backend/database/seed.go]: @krismp done kk https://github.com/nodefluxio/weber/pull/130/commits/f8d9ec9a918714f61f7bd05ad07a2e86be521cba

**Review by lazuardyk — APPROVED** [review]
mantaps

File: backend/database/seed.go (hunk 1 above)
Comment (krismp) [reply backend/database/seed.go]: Luar biasa sekali sih antum

**Review by krismp — APPROVED** [review]

File: backend/database/seed.go (hunk 1 above)
Comment (gregoriusjimmy) [reply backend/database/seed.go]: GOKIL Nanda

**Review by asyafalni — APPROVED** [review]

**Pull request merged**